scenes/src/app/scenes/button/button-scene.scss

```diff
@@ -0,0 +1,39 @@
+.buttons {
+  position: relative;
+  width: 50px;
+  top: 20px;
+}
+
+button {
+  margin: 15px !important;
+  width: 100px;
+  height: 50px;
+}
+
+#left {
+  position: absolute;
+  left: 80px;
+}
+
+#right {
+  position: absolute;
+  left: 260px;
+}
+
+.mat-button-wrapper {
+  font-size: 15px;
+}
+
+.mat-raised-button[disabled][disabled] {
```

Comment:
Why `[disabled][disabled]`? Is that a mistake? If it's needed, please add a comment.

Reply:
Managed to remove them by changing the theme!